Docs.Site/Docs/Walkthrough/Getting-Started-with-Ruby-Gems.md

```diff
@@ -0,0 +1,94 @@
+		<gem_name>-<gem_version>.gem
+* **Package metadata.** When you upload a package, MyGet will import most of its metadata from the .gemspec file. If your package does not include a .gemspec file, you will see much less information about that package displayed in your MyGet feed.
+Include Dependencies. You can also include dependencies when you upload packages from your local machine. If you choose to “Include dependencies,” MyGet will parse the .gemspec file in your gem to gather a list of all needed dependencies and try to fetch them from RubyGems.org. If MyGet cannot find the specified dependencies in the upstream source, only your primary gem will be added. You can always manually add dependencies from RubyGems.org or upload from your local machine later.
+* **Mirror Dependencies:** If you choose to mirror dependencies, MyGet will attempt to import the gem dependencies from RubyGems.org and host them for you on your MyGet feed. Otherwise, MyGet will add references to the dependencies on RubyGems.org so that you can view them in your MyGet feed without counting against your MyGet storage allotment.
+
+
+2. (optional) **Enable upstream source proxying for RubyGems.org** to seamlessly blend your MyGet feed with the RubyGems.org repository.
+
+Upstream package sources should be considered as *upstream repositories* for your MyGet feed. You can *aggregate* them in a single MyGet feed, filter each of them individually, and even *proxy* upstream package sources to include them in your feed queries.
+
+This way, you can make packages from multiple community package hosting sites (like RubyGems.org, NuGet.org, npmjs.org, etc.) directly available via your MyGet feed, whether or not you have previously added them. 
+
+(For more information about using Upstream Sources, check our <a href="https://docs.myget.org/docs/reference/upstream-sources" target="_blank" rel="noopener">help docs</a>.)
+
+From your new Feed page, click the *Upstream Sources* tab in the menu to the left. Click the Edit for the RubyGems *upstream source* displayed in the preconfigured upstream sources list, and enable the*Make all upstream packages available in clients* option. If you prefer to have the package binaries downloaded to your feed when making a request for an upstream package not currently added to your MyGet feed, also enable the *Automatically add downloaded upstream packages to the current feed (mirror) option.* 
+
+![Proxy upstream sources into MyGet.](/docs/walkthrough/Images/gem_docs_edit_upstream.jpg)
+
+## Pushing a gem package to your feed from the command line
+
+You can easily upload a gem package through the MyGet.org website, but you can also push a gem from your command line if you would like using the cUrl utility and API key from your MyGet account. 
+		curl -k -X POST https://<your_myget_domain>/F/<your_feed_name>/gem/upload -H “Authorization: Bearer <your_api_key>” -F “data=@<gem_name>-<gem_version>.gem”
```

Comment:
@kacper-r the "code formatting" I attempted here is incorrect - I will try to update

Reply:
I was able to fix